### DISCUSSION TITLE
欢迎来到世界

### DISCUSSION THREAD

**Discussion by Novicexs** [share121/inter-knot#911] [其他]
Hello Word
![176542181B7BCBA95F4731C8282CE268](https://github.com/user-attachments/assets/29e3cf31-5f72-4eed-bdcf-ce542b7ab491)

**Comment by gaojiaobei** [comment]
欸？我吗？

**Comment by nailbiting21** [comment]
cout<<"Hello world"<<endl;

**Comment by Wu-jian-NGX-zhanghao** [comment]
print ：”hello ward“
print ：”TD“

**Comment by lingyushen** [comment]
01001000 01100101 01101100 01101100 01101111 00101100 00100000 01010111 01101111 01110010 01101100 01100100 00100001

**Comment by Xmug** [comment]
去六分街要走光映大道

**Comment by AetherLude** [comment]
你好，文字？

**Comment by LHk416** [comment]
六分街？*走二仙桥！*

**Comment by fairy-B** [comment]
嘿！那可太经典了

**Comment by xinaozi** [comment]
太好了!bu~bu~

**Comment by Buer-Nahida** [comment]
```rust
use std::io::{self, Write};

/// Print "Hello, world!" and change to next line.
/// Same as this.
/// ```
/// println!("Hello, world!");
/// ```
fn main() {
    io::stdout()
        .write_all("Hello, world!\n".as_bytes())
        .unwrap();
}
```

**Comment by lllliufeng** [comment]
```C#
using System;
namespace helloWord
{
    class helloWordClass
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello Word!");
            Console.ReadKey();
        }
    }
}

**Comment by capacitor1** [comment]
Hello World
你好，世界
ハローワールド
Salut tout le monde
Всем привет
SGVsbG8gV29ybGQ=
JBSWY3DPEBLW64TMMQ======
48656C6C6F20576F726C64
01001000 01100101 01101100 01101100 01101111 00100000 01010111 01101111 01110010 01101100 01100100
<~87cURD]i,"Ebo7~>
b10a8db164e0754105b7a99be72e3fe5
呢嗯？嗯嗯嗯嗯嗯呢？呢呢嗯嗯嗯嗯呢？！嗯嗯嗯嗯嗯呢？！嗯嗯嗯嗯嗯呢？！！嗯嗯嗯嗯嗯？嗯嗯嗯嗯嗯嗯呢呢呢！嗯嗯嗯嗯呢？！！嗯嗯嗯嗯呢！嗯？嗯嗯嗯嗯呢？！嗯嗯嗯嗯嗯呢？呢嗯嗯嗯嗯嗯

doge

**Comment by cpu644** [comment]
`print("Hello World!")`

**Comment by su2569** [comment]
原文: Hello Word
base64: SGVsbG8gV29yZA==
md5: ed0a96e83ab7b0910fcbcc131b2e6b82
md5_middle: 3ab7b0910fcbcc13
md5(md5($pass)): 5801935884dc87c9f9a6f13f37924076
md5(md5(md5($pass))): 2c2b0f8928c5ec0ef1d3ede7d4f1eaca
md5(unicode): eb3152ad9fa0c9749a7506db65782f89
md5(base64): 7QqW6Dq3sJEPy8wTGy5rgg==
mysql: 28c0edd271edade4
mysql5: b204c4e91e96470b0eba2fa97888156ee319ad65
ntlm: ae957577aa7641ec2118565bd0b94272
sha1: 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad
sha1(sha1($psss)): 1deffc52101264a16227c7975aabe05e5040ab95
sha1(md5($psss)): 590b8868fd96763beafbdb61b2f9534e981e4ead
md5(sha1($psss)): dba4469a96ba8328ef5134d4d28d995d
sha256: 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0
sha256(md5($pass)): 050a77c4de5ff4fa8bb92bbfc89458e2319ea62dd153a619eccf5773ed57ab6c
sha384: 1e40aa579d1c826d3c48370084698e9f0da212aa615791ed9f851673de7a959124627445ef061bd93edd5dd94ece1b54
sha512: 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2
md5(md5($pass).$salt);VB;DZ: 5801935884dc87c9f9a6f13f37924076
md5($pass.$salt): ed0a96e83ab7b0910fcbcc131b2e6b82
md5($salt.$pass): ed0a96e83ab7b0910fcbcc131b2e6b82
md5($salt.$pass.$salt): ed0a96e83ab7b0910fcbcc131b2e6b82
md5($salt.md5($pass)): 5801935884dc87c9f9a6f13f37924076
md5(md5($salt).$pass): f87d03c22a04da73c44b1ea4b4b9dd96
md5($pass.md5($salt)): ced0922619b2fbaea9473bc3ba372601
md5(md5($salt).md5($pass)): 3e45710e78756f5826ff7be1c1444813
md5(md5($pass).md5($salt)): c9da009814fdc56ab19cbcad510980c5
md5(substring(md5($pass),8,16)): 70f00c72c95952c75767fa9a38f287e3
sha1($pass.$salt): 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad
sha1($salt.$pass): 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad
sha256($pass.$salt): 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0
sha256($salt.$pass): 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0
sha512($pass.$salt): 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2
sha512($salt.$pass): 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2
MSSQL2015: 0x01000000000041ebf8ccea7138f538a94a1f42d8b988dbf5c618

**Comment by su2569** [comment]
> 原文: Hello Word base64: SGVsbG8gV29yZA== md5: ed0a96e83ab7b0910fcbcc131b2e6b82 md5_middle: 3ab7b0910fcbcc13 md5(md5($pass)): 5801935884dc87c9f9a6f13f37924076 md5(md5(md5($pass))): 2c2b0f8928c5ec0ef1d3ede7d4f1eaca md5(unicode): eb3152ad9fa0c9749a7506db65782f89 md5(base64): 7QqW6Dq3sJEPy8wTGy5rgg== mysql: 28c0edd271edade4 mysql5: b204c4e91e96470b0eba2fa97888156ee319ad65 ntlm: ae957577aa7641ec2118565bd0b94272 sha1: 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad sha1(sha1($psss)): 1deffc52101264a16227c7975aabe05e5040ab95 sha1(md5($psss)): 590b8868fd96763beafbdb61b2f9534e981e4ead md5(sha1($psss)): dba4469a96ba8328ef5134d4d28d995d sha256: 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0 sha256(md5($pass)): 050a77c4de5ff4fa8bb92bbfc89458e2319ea62dd153a619eccf5773ed57ab6c sha384: 1e40aa579d1c826d3c48370084698e9f0da212aa615791ed9f851673de7a959124627445ef061bd93edd5dd94ece1b54 sha512: 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2 md5(md5($pass).$salt);VB;DZ: 5801935884dc87c9f9a6f13f37924076 md5($pass.$salt): ed0a96e83ab7b0910fcbcc131b2e6b82 md5($salt.$pass): ed0a96e83ab7b0910fcbcc131b2e6b82 md5($salt.$pass.$salt): ed0a96e83ab7b0910fcbcc131b2e6b82 md5($salt.md5($pass)): 5801935884dc87c9f9a6f13f37924076 md5(md5($salt).$pass): f87d03c22a04da73c44b1ea4b4b9dd96 md5($pass.md5($salt)): ced0922619b2fbaea9473bc3ba372601 md5(md5($salt).md5($pass)): 3e45710e78756f5826ff7be1c1444813 md5(md5($pass).md5($salt)): c9da009814fdc56ab19cbcad510980c5 md5(substring(md5($pass),8,16)): 70f00c72c95952c75767fa9a38f287e3 sha1($pass.$salt): 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad sha1($salt.$pass): 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad sha256($pass.$salt): 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0 sha256($salt.$pass): 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0 sha512($pass.$salt): 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2 sha512($salt.$pass): 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2 MSSQL2015: 0x01000000000041ebf8ccea7138f538a94a1f42d8b988dbf5c618

.

**Comment by su2569** [comment]
> 原文: Hello Word base64: SGVsbG8gV29yZA== md5: ed0a96e83ab7b0910fcbcc131b2e6b82 md5_middle: 3ab7b0910fcbcc13 md5(md5($pass)): 5801935884dc87c9f9a6f13f37924076 md5(md5(md5($pass))): 2c2b0f8928c5ec0ef1d3ede7d4f1eaca md5(unicode): eb3152ad9fa0c9749a7506db65782f89 md5(base64): 7QqW6Dq3sJEPy8wTGy5rgg== mysql: 28c0edd271edade4 mysql5: b204c4e91e96470b0eba2fa97888156ee319ad65 ntlm: ae957577aa7641ec2118565bd0b94272 sha1: 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad sha1(sha1($psss)): 1deffc52101264a16227c7975aabe05e5040ab95 sha1(md5($psss)): 590b8868fd96763beafbdb61b2f9534e981e4ead md5(sha1($psss)): dba4469a96ba8328ef5134d4d28d995d sha256: 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0 sha256(md5($pass)): 050a77c4de5ff4fa8bb92bbfc89458e2319ea62dd153a619eccf5773ed57ab6c sha384: 1e40aa579d1c826d3c48370084698e9f0da212aa615791ed9f851673de7a959124627445ef061bd93edd5dd94ece1b54 sha512: 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2 md5(md5($pass).$salt);VB;DZ: 5801935884dc87c9f9a6f13f37924076 md5($pass.$salt): ed0a96e83ab7b0910fcbcc131b2e6b82 md5($salt.$pass): ed0a96e83ab7b0910fcbcc131b2e6b82 md5($salt.$pass.$salt): ed0a96e83ab7b0910fcbcc131b2e6b82 md5($salt.md5($pass)): 5801935884dc87c9f9a6f13f37924076 md5(md5($salt).$pass): f87d03c22a04da73c44b1ea4b4b9dd96 md5($pass.md5($salt)): ced0922619b2fbaea9473bc3ba372601 md5(md5($salt).md5($pass)): 3e45710e78756f5826ff7be1c1444813 md5(md5($pass).md5($salt)): c9da009814fdc56ab19cbcad510980c5 md5(substring(md5($pass),8,16)): 70f00c72c95952c75767fa9a38f287e3 sha1($pass.$salt): 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad sha1($salt.$pass): 44d3a83db52a0cf0f908ad0ef55c83f4771f66ad sha256($pass.$salt): 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0 sha256($salt.$pass): 9aea7eb32aebe6f10c3ec1381c66e18ffd50f20c54e29fd85b5ae159cf80c9d0 sha512($pass.$salt): 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2 sha512($salt.$pass): 6d01431023e996f7234c8db3ca10d7e341e397595f79b9a3063d2297ddb8021bde9f50839d94c4c802a45e7c3f6ed66ff22ef32874134eb6ca382b59ba8d2bf2 MSSQL2015: 0x01000000000041ebf8ccea7138f538a94a1f42d8b988dbf5c618

.

**Comment by su2569** [comment]
6

**Comment by TXGY-INTEL** [comment]
printf("Hello world!");

**Comment by wangliuzhe** [comment]
铃妹可爱捏
